client/src/components/BackupCodes/Login.scss

```diff
@@ -0,0 +1,10 @@
+.mfa-login-backup-codes {
+  &__container {
+    display: grid;
+    grid-template-columns: 50% 50%;
```

Comment:
Are there any polyfills we need to support CSS grids?

Reply:
No. There are some minor caveats with IE10 but this usage is fine (I think)